model/pod/pod.go

```diff
@@ -691,7 +712,8 @@ func (p *Pod) SetRunningTask(ctx context.Context, env evergreen.Environment, tas
 		return errors.New("pod was not updated")
 	}
 
-	p.RunningTask = taskID
+	p.TaskRuntimeInfo.RunningTaskID = taskID
```

Comment:
We can assign the whole `TaskRuntimeInfo` here for clarity rather than setting both fields individually since the whole struct is just a task Id and execution.

Reply:
Ok, sgtm